osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json

```diff
@@ -40,6 +40,130 @@
         }
       ]
     },
+    {
+      "tag": "ACTIVE_ENERGY_IMPORT_RATE_1",
+      "description": "Active energy import (+A) rate 1",
+      "note": null,
+      "class-id": 3,
+      "version": 0,
+      "obis": "1.0.1.8.1.255",
+      "group": "ELECTRICITY",
+      "meterTypes": [
+        "SP",
+        "PP"
+      ],
+      "attributes": [
+        {
+          "id": 2,
+          "description": "value",
+          "datatype": "double-long-unsigned",
+          "valuetype": "DYNAMIC",
+          "value": "0",
```

Comment:
value staat niet op 0 in de spec, maar er staat in de spec: "Rate 1 = Low tariff"

Reply:
The value of this register holds the meter value for rate 1, not the value "Rate 1 = Low tariff". This is unclear in the P3 document. Since the meter value is dynamic (it will be updated by te meter), the value need to be removed.